content/en/docs/concepts/security/index.md

```diff
@@ -137,6 +137,55 @@ Istio provisions keys and certificates through the following flow:
 1. Istio agent monitors the expiration of the workload certificate.
    The above process repeats periodically for certificate and key rotation.
 
+## ClusterTrustBundle 
+
+ClusterTrustBundle is a Kubernetes Custom Resource Definition(CRD) introduced to help manage trusted Certificate Authority (CA) bundles cluster-wide.It`s primarily used to distribute and trust public  X.509 certificates across the entire cluster.This concept is especifically useful in environments where components and workloads need to validate TLS certificates signed by non-standard or private CAs.Istio has added experimental support for this in recent version, making it easier to manage trust for services.
+
+#### Enabling the feature 
+
+To use ClusterTrustBundle in Istio, you must enable it by setting a flag during installation.
+Here`s how:
+<ul>
+   <li> Add this to your istio configuration</li>
+</ul>
+
+{{< text yaml >}}
+values:
+  pilot:
+    env:
+      ENABLE_CLUSTER_BUNDLE_API: "true"
+{{< /text >}}
+
+<ul>
+   <li>Ensure your Kubernetes cluster is version 1.27 or later, as ClusterTrustBundle was introduced then.
```

Comment:
ClusterTrustBundles must be activated through a feature flag, even in current k8s versions